docs/how-to-use-customized-configuration.md

```diff
@@ -0,0 +1,245 @@
+
+## Mount arbitrary files like key and certificate files to the container in Scalar DL Helm Charts
+
+You can mount any files to the container when you use the Scalar DL Helm Charts. For example, you need to mount the key and certificate files to run the Scalar DL Auditor.
+
+* Configuration example
+    * Scalar DL Ledger
+      ```yaml
+      ledger:
+        ledgerProperties: |
+          ...
+          scalar.dl.ledger.proof.enabled=true
+          scalar.dl.ledger.auditor.enabled=true
+          scalar.dl.ledger.proof.private_key_path=/keys/private-key
+      ```
+    * Scalar DL Auditor
+      ```yaml
+      auditor:
+        auditorProperties: |
+          ...
+          scalar.dl.auditor.cert_path=/keys/certificate
+          scalar.dl.auditor.private_key_path=/keys/private-key
+      ```
+
+In this example, you need to mount a **private-key** and a **certificate** file under the `/key` directory in the container. And, you need to mount files named `private-key` and `certificate`. You can use `extraVolumes` and `extraVolumeMounts` to mount these files.
+
+1. Set `extraVolumes` and `extraVolumeMounts` in the custom values file using the same syntax of Kubernetes manifest. You need to specify the directory name to the key `mountPath`.
+   * Example
+        * Scalar DL Ledger
+          ```yaml
+          ledger:
+            extraVolumes:
+              - name: ledger-keys
+                secret:
+                  secretName: ledger-keys
+            extraVolumeMounts:
+              - name: ledger-keys
+                mountPath: /keys
+                readOnly: true
+          ```
+       * Scalar DL Auditor
+         ```yaml
+         auditor:
+            extraVolumes:
+              - name: auditor-keys
+                secret:
+                  secretName: auditor-keys
+            extraVolumeMounts:
+              - name: auditor-keys
+                mountPath: /keys
+                readOnly: true
+          ```
+
+1. Create a `Secret` resource that includes key and certificate files.  
+   You need to specify the file name as keys of `Secret`.
+   * Example
+       * Scalar DL Ledger
+         ```console
+         kubectl create secret generic ledger-key-secret \
+           --from-file=private-key=./ledger-key.pem
+         ```
+       * Scalar DL Auditor
+         ```console
+         kubectl create secret generic auditor-key-secret \
+           --from-file=private-key=./auditor-key.pem \
+           --from-file=certificate=./auditor-cert.pem
+         ```
+
+1. Deploy Scalar products with the above custom values file.  
+   After deploying Scalar products, key and certificate files are mounted under the `key` directory as follows.
+   * Example
+       * Scalar DL Ledger
+         ```console
+         $ ls -l /keys/
+         total 0
+         lrwxrwxrwx 1 root root 18 Jun 27 03:12 private-key -> ..data/private-key
```

Comment:
Is `..data/private-key` displayed as it is or does the `..` represents an abbreviation? If an abbreviation, I think something like `.../data/private-key` or `/path/to/data/private-key` is more appropriate.

Reply:
At a first look `..` seemed somewhat strange to me, but I understand that it is part of a real (hidden) file name. Thank you!